Dynatrace/dynatrace-operator
# Cleanup Dynakube's config and enrichment secrets when deleting it

## Description

https://dt-rnd.atlassian.net/browse/DAQ-2266

Currently, `dynatrace-dynakube-config` and `dynatrace-metadata-enrichment-endpoint` secrets doesn't get deleted when the Dynakube gets deleted.

This PR adds the cleanup of those.

## How can this be tested?

- On main branch: deploy a DK and delete it, check that these secrets doesn't get deleted
- On this branch: do the same and check that they get deleted
